core/modules/views_ui/views_ui.theme.inc

```diff
@@ -4,15 +4,23 @@
  * Theme functions for the Views UI module.
  */
 
+/**
+ * Display the view name and its machine name.
+ */
+function theme_views_ui_view_name($variables) {
+  $view = $variables['view'];
+
+  $output = '';
+  $output .= '<div class="views-ui-view-name">' . check_plain($view->get_human_name()) . '</div>';
+  $output .= '<div class="views-ui-view-machine-name">(</span>' . check_plain($view->name) . '</span>)</div>';
```

Comment:
```suggestion
  $output .= '<div class="views-ui-view-machine-name">(<span>' . check_plain($view->name) . '</span>)</div>';
```

Reply:
Looks like this mistake is still in here?